microsoft-edge/devtools-guide-chromium/device-mode/index.md

```diff
@@ -64,17 +66,34 @@ In the following figure, the width is set to `400` and the height is set to `736
 
 If you need more space on your screen, you can always change the position of DevTools as explained in [Change DevTools placement (Undock, Dock to bottom, Dock to left)](/microsoft-edge/devtools-guide-chromium/customize/placement).
 
-#### Show media queries
 
-If you have defined media queries on your page, jump to the viewport dimensions where those media queries take effect by showing media query breakpoints above your viewport.  Select **More options** > **Show media queries**.
+<!-- ---------- -->
+###### Show media queries
+
+A _media query_ defines a breakpoint, which is a browser viewport width.  A webpage can define a responsive layout for each breakpoint (width) that it defines.
+
+To show media query breakpoints above the viewport:
+
+1. Click the **More options** (![More Options icon](./index-images/more-options-icon.png)) button, and then select **Show media queries**:
+
+   ![Show media queries](./index-images/device-toolbar-more-options-show-media-queries.png)
+   <!-- https://microsoftedge.github.io/Demos/devtools-a11y-testing/ -->
+
+   If `max-width` or `min-width` breakpoints are defined by the webpage, DevTools displays additional bars above the viewport:
+   * A blue bar with `max-width` breakpoints.
+   * An orange bar with `min-width` breakpoints:
```

Comment:
The green bar corresponds to media queries that use both `min-width` and `max-width` at the same time (to define a range).

Reply:
Incorp'd.  Can Resolve thread.